### PR TITLE
feat(PDEV-1944) get form values

### PR DESCRIPTION
**Pull request by yuan616510**

### CODE REVIEW

**Comment by yuan616510** [comment]
screen shot 2019-01-29 at 9 47 57 am

**Comment by yuan616510** [comment]
Ready for review.

**Comment by bondricek** [comment]
the schema should not be pershing specific, can add the properties from the abstract class and allow additional properties

**Comment by yuan616510** [comment]
This is ready for review again.

File: api/v10/examples/getAccountOpeningFormValues.res.json [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  "bankAccountNumber": "*****1234",
+  "abaRoutingNumber": "*****1234",
+  "bankAccountTitle": "bankAccountTitle",
+  "bankCity": "bankCity",
+  "bankState": "bankState",
+  "bankPhone": "bankPhone",
+  "externalFirmId": "externalFirmId",
+  "accountFirmTrizicId": "accountFirmTrizicId",
+  "registrationType": "registrationType"
+}
```
Comment (kongmalin) [inline api/v10/examples/getAccountOpeningFormValues.res.json]: newline

File: api/v10/examples/getAccountOpeningFormValues.res.json [hunk 2]
```diff
@@ -0,0 +1,120 @@
+{
+  "holders": [{
+    "type": "type",
+    "code": "code",
+    "firstName": "****",
+    "middleName": "****",
+    "lastName": "****",
+    "dateOfBirth": "****",
+    "fullName": "****",
```
Comment (kongmalin) [inline api/v10/examples/getAccountOpeningFormValues.res.json]: Can move fullName to above dateOfBirth to be the same as definition

File: api/v10/examples/getAccountOpeningFormValues.res.json [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    "setMailingToLegal": true,
+    "mailingAddressLine1": "****ine1",
+    "mailingAddressLine2": "****ine2",
+    "mailingAddressLine3": "****ine3",
+    "mailingCity": "mailingCity",
+    "mailingState": "mailingState",
+    "mailingZipCode": "mailingZipCode",
+    "mailingCountry": "mailingCountry"
+  }, {
+    "type": "type",
```
Comment (kongmalin) [inline api/v10/examples/getAccountOpeningFormValues.res.json]: This holder has no code?

File: api/v10/examples/getAccountOpeningFormValues.res.json [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+    "fullLegalStreetAddress": "****ress",
+    "legalCity": "legalCity",
+    "legalState": "legalState",
+    "legalZipCode": "legalZipCode",
+    "legalCountry": "legalCountry",
+    "email": "p******@123.com",
+    "homePhone": "******4567",
+    "cellPhone": "******4567"
+  }, {
+    "type": "type",
```
Comment (kongmalin) [inline api/v10/examples/getAccountOpeningFormValues.res.json]: This also has no code?

**Comment by yuan616510** [comment]
Comments addressed

**Pull request merged**